npm-shrinkwrap.json

```diff
@@ -5982,6 +6017,11 @@
       "from": "sax@>=0.6.0",
       "resolved": "https://registry.npmjs.org/sax/-/sax-1.2.2.tgz"
     },
+    "seek-bzip": {
```

Comment:
Hmm, not sure – looks like streaming isn't implemented in `seek-bzip`, so we're probably better off waiting for an alternative for bzipping things.